### ISSUE TITLE
Include intro exercises/information to ensure base level knowledge

### ISSUE THREAD

**Issue by tylerauerbeck** [rht-labs/enablement-docs#483]
As part of do500 we introduce a lot of information. At times we can forget that not everyone may be familiar with the terminology or commands that we assume that they will know. To ensure that we have a common set of knowledge when starting the course we should provide a set of information/exercises that will allow students to familiarize themselves with that content (intro to git, basic linux commands, etc.)

**Comment by oybed** [comment]
This is somewhat currently in the beginning portion of the Tech write-up - i.e.:  under the "Home" left-side menu. However, we should spend some time on ensure it covers the proper areas + figure out how to best allow time to go through this - either as a mob, individually or as a presentation.

**Comment by tylerauerbeck** [comment]
Agreed. I think it's just a matter deciding what we might want to include.

Potentially related: https://github.com/rht-labs/enablement-ci-cd/issues/17
